### PR TITLE
fix: add aria-label for a11y

### PR DESCRIPTION
**Pull request by bigint**
Part of #1240

This PR will fix this error

Screenshot 2021-12-12 at 7 32 11 PM

### CODE REVIEW

File: src/components/Topnav.vue [hunk 1]
```diff
@@ -101,6 +101,7 @@ onMounted(() => setTitle());
               v-if="!$auth.isAuthenticated.value"
               @click="modalAccountOpen = true"
               :loading="loading || web3.authLoading"
+              aria-label="Login"
```
Comment (bonustrack) [inline src/components/Topnav.vue:104]: It should be connect wallet with translation
```suggestion
              :aria-label="$t('connectWallet')"
```

File: src/components/Topnav.vue [hunk 2]
```diff
@@ -123,6 +124,7 @@ onMounted(() => setTitle());
               v-if="!domain"
               @click="toggleSkin"
               class="float-right ml-2"
+              aria-label="Toggle Skin"
```
Comment (bonustrack) [inline src/components/Topnav.vue:127]: This should use a translation message, and we should use "Sentence case" instead of "Title Case", (Toggle skin)

Comment (bigint) [reply src/components/Topnav.vue]: @bonustrack Just updated it, I'm not sure, should I add it only to the `default.json` file or all files in locales?

Comment (samuveth) [reply src/components/Topnav.vue]: @bigint just he default.json

**Comment by samuveth** [comment]
Should this also be added to the Twitter button?

**Comment by bigint** [comment]
@samuveth Twitter is not button, its a link and it don't need aria-label 🙇🏻

**Review by samuveth — APPROVED** [review]
LGTM

**Pull request merged**